src/lsm/manifest_level.zig

```diff
@@ -110,6 +216,11 @@ pub fn ManifestLevelType(
             if (table.visible(lsm.snapshot_latest)) level.table_count_visible += 1;
             level.generation +%= 1;
 
+            level.range.update(.{ .include = KeyRange{
+                .key_min = table.key_min,
+                .key_max = table.key_max,
+            } });
```

Comment:
Since this is control plane, it would be good to assert that the level range has been updated correctly.

Reply:
Added to assertion to check validity of ManifestLevel key range as part of https://github.com/tigerbeetle/tigerbeetle/pull/944/commits/c2ee1b25726e53fe752f37bdadb21772a72d95f8.